src/lib/fillers/fetchCollectorSignals.ts

```diff
@@ -0,0 +1,118 @@
+import { isFeatureFlagEnabled } from "lib/featureFlags"
```

Comment:
is the folder name `fillers` correct? or is it supposed to be `filters`?

Reply:
So I put this in the folder next to its neighbor at the point it is called - [enrichArtworkWithPriceInsights()](https://github.com/artsy/metaphysics/blob/537f2d7155c41b775240dc11e33056ed75502ce0/src/lib/fillers/enrichArtworksWithPriceInsights.ts#L8-L26). These are the only two files in that folder.

However it doesn't really enrich anything any more and is really more of a complex loader. I could put it in the context - like [creating all the basic loaders](https://github.com/artsy/metaphysics/blob/a755fd94e5f1afd87cccce1e2d05f4cae2ce8940/src/lib/loaders/index.ts#L73-L96) and then constructing this last one from it.

As a first step I'm just moving it into lib/loaders. If we want to make this a first-class member of the context I can do that as well.